### PR TITLE
Add docker compose definition to run the relayer as a service

### PR DESCRIPTION
**Pull request by sameersubudhi**
### What was the problem?

This PR resolves #LISK-960

### How was it solved?

- [x] Optimized Dockerfile
- [x] Added docker-compose.yml
- [x] Update `appspec.yml` to use docker-compose.yml
- [x] Set up new CodePipeline & CodeDeploy application to deploy to the new EC2 server instance
- [x] Set up new CloudWatch group to capture the docker container logs
- [x] Add scripts for local development on docker
- [x] Remove unused file: `cloudbuild.yml`

### How was it tested?

Against Sepolia on the new server instance against the current branch.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      retries: 3
+
+  relayer_1:
+    image: ${AWS_ECR_REGISTRY}/${AWS_ECR_REPOSITORY}:${ACROSS_RELAYER_IMAGE_TAG}
+    platform: linux/amd64
+    depends_on:
+      redis:
+        condition: service_healthy
+    volumes:
+      - ./scripts/lisk/docker/${NETWORK:-sepolia}/containerStartBot_1.sh:/home/lisk/across-relayer/scripts/start.sh
```
Comment (Incede) [inline docker-compose.yml:24]:
```suggestion
      - ./scripts/lisk/docker/${NETWORK:-mainnet}/containerStartBot_1.sh:/home/lisk/across-relayer/scripts/start.sh
```
Should it be `mainnet` (assuming this PR goes onto the production server) ?

**Review by Incede — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (sameersubudhi) [reply docker-compose.yml:24]: I'll update it to `mainnet`. But leaving it `sepolia` also shouldn't be an issue as the `NETWORK` environment variable is set from the appropriate AWS secrets within L19 of `scripts/lisk/docker/setEnvVariables.sh`.

File: scripts/lisk/docker/setEnvVariables.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -eu
+
+# Set env var from secrets
+# secret_id=arn:aws:secretsmanager:eu-west-3:132202091885:secret:mainnet/lisk-across-relayer/aws-CSi7ka
+secret_id=arn:aws:secretsmanager:eu-west-3:132202091885:secret:sepolia/across-relayer-dev/aws-7CIqpl
```
Comment (matjazv) [inline scripts/lisk/docker/setEnvVariables.sh:6]: What about here? Should we use `mainnet` instead of `sepolia`?

**Review by matjazv — CHANGES_REQUESTED** [review]

File: scripts/lisk/docker/setEnvVariables.sh (hunk 2 above)
Comment (sameersubudhi) [reply scripts/lisk/docker/setEnvVariables.sh:6]: Yes, as mentioned earlier on Slack, I'll finally update this and the workflow file in the final commit before merging the PR.
Setting it to `mainnet` currently will run another competing instance of the bot on the mainnet :wink:

**Comment by matjazv** [comment]
Newly added scripts are not executable files (`chmod 755`).

**Review by matjazv — APPROVED** [review]
Tested locally and it works!

**Review by Nazgolze — APPROVED** [review]

**Review by Incede — APPROVED** [review]

**Review by matjazv — APPROVED** [review]

**Pull request merged**